packages/gateway/.env.example

```diff
@@ -0,0 +1 @@
+PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Comment:
I think we can leave it blank or with messages like "Fill in your private key" so people following the instruction would not accidentally use this private key and assume this is the private key to the signing address(`0x3B7D34d0E7e807A9D7aD74F094C5379aca61460D`) in readme

Reply:
OK! I thought this was indeed better! Will be updated today!